app/ai/voice/agents/breeze_buddy/template/global_function.py

```
@@ -0,0 +1,318 @@
+                f"{func.description} {GLOBAL_FUNCTION_DESCRIPTION_SUFFIX}"
+            )
+
+        logger.debug(
+            f"Building HTTP global function: {func.name}, "
+            f"http_request={func.http_request.method.value} {func.http_request.url}"
+        )
+
+        # Create outer wrapper that passes function_config via kwargs
+        # This follows the same pattern as normal functions passing transition_to, hooks, etc.
+        def create_wrapper(captured_func: GlobalHttpFunction):
+            async def wrapper_handler(llm_args, flow_manager):
```

Ruff (0.14.13)

Unused function argument: `flow_manager`

(ARG001)